components/Intro.vue

```diff
@@ -18,17 +18,16 @@
         <p>nwPlus Presents</p>
         <img src="~@/assets/lhd.svg" class="lhd">
         <p>@ University of British Columbia</p>
-        <p
-          v-for="item in text"
-          :key="item"
-          class="date"
-        >
+        <p v-for="item in text" :key="item" class="date">
           {{ item }}
         </p>
         <Button title="Contact Us" url="mailto:lhd@nwplus.io" />
-        <Button disabled title="Sign Up" url="#" />
+        <Button
+          title="Sign Up"
+          url="https://organize.mlh.io/participants/events/1610-ubc-lhd-2019-learn-day"
+        />
```

Comment:
any particular reason why the <p> tag was put into one line but the <button> was broken up into many lines? just curious